servicetest/dbus/dbusapp.py

```diff
@@ -65,13 +65,21 @@ def run(self):
         self.service = Service(bus)
         self.__loop.run()
 
-    def wait_until_requests(self, multiplier=1, timeout=1):
+    def wait_until_requests(self, multiplier=1, timeout=5):
```

Comment:
This will break previous work with lowering the execution time. There should be a real solution to this problem.